ci-operator/config/redhat-appstudio/rhtap-cli/redhat-appstudio-rhtap-cli-main.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  rhtap-installer:
+    name: redhat-appstudio
+    namespace: ci
+    tag: rhtap-installer
+build_root:
+  project_image:
+    dockerfile_path: ci/Dockerfile
+images:
+- dockerfile_path: ci/Dockerfile
+  from: src
+  to: rhtap-cli
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: rhtap-cli-install
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-2
+    owner: rhtap
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.14"
```

Comment:
I think we need to use the latest version of OCP `4.15`

Reply:
We don't have a cluster pool for 4.15 but we could create one and depracate our cluster pool for 4.14 what do you think 

@flacatus ^^

https://redhat-internal.slack.com/archives/C065GQMTLN4/p1719324110909029

Reply:
submitted pr to add a 4.15 pool here https://github.com/openshift/release/pull/54224